lib/mysql.js

```diff
@@ -200,17 +200,7 @@ MySQL.prototype.executeSQL = function(sql, params, options, callback) {
   }
 };
 
-/**
- * Update if the model instance exists with the same id or create a new instance
- *
- * @param {String} model The model name
- * @param {Object} data The model instance data
- * @param {Function} [callback] The callback function
- */
-MySQL.prototype.updateOrCreate = MySQL.prototype.save =
-  function(model, data, options, callback) {
-    var fields = this.buildFields(model, data);
-
+ MySQL.prototype.modifyOrCreate = function(model, data, options, fields ,callback) {
```

Comment:
the comma is still in the wrong spot (there is a space in front of it, when there should only be one after)